SPECS/core-packages/core-packages.spec

```diff
@@ -1,90 +1,81 @@
-Name:           core-packages
 Summary:        Metapackage with core sets of packages
+Name:           core-packages
 Version:        0.1
-Release:        21%{?dist}
+Release:        22%{?dist}
 License:        ASL 2.0
-Group:          System Environment/Base
-URL:            http://aka.ms/mariner
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          System Environment/Base
+URL:            https://aka.ms/mariner
 
 %description
 Metapackage holding sets of core packages for different applications.
 
-%package base-image
```

Comment:
>base-image [](start = 9, length = 10)

This removal of `%package base-image` doesn't look right. We still have the separate `Summary` tag and `%files` section for it left. Could you explain what we're doing here?